poutyne/framework/callbacks/wandb_logger.py

```diff
@@ -0,0 +1,256 @@
+import os
+import warnings
+from typing import Dict, Optional, List
+
+import torch
+
+
+from . import Logger
+
+try:
+    import wandb
+
+except ImportError:
+    wandb = None
+
+
+class WandBLogger(Logger):
+    """
+    WandB logger to manage logging of experiments
+    parameters, metrics update, models log, gradient
+    values and other information. The logger will log
+    all run into the same experiment.
+    Args:
+        name(str): Display name for the run.
+        group (Optional[str]): the name of the group to which this run belongs.
```

Comment:
```suggestion
        group (Optional[str]): the group's name to which this run belongs.
```

Reply:
Is it the "name to which this runs belongs" or the "group to which this runs belongs" ?
```suggestion
        group (Optional[str]): The name of the group to which this run belongs.
```

Reply:
I changed it for the real wandb definition : "Specify a group to organize individual runs into a larger experiment"